### PR TITLE
CLDC-2247 Add locations filter

### PR DESCRIPTION
**Pull request by kosiakkatrina**
Add status filter for locations:
image

### CODE REVIEW

File: app/helpers/filters_helper.rb [hunk 1]
```diff
@@ -56,6 +56,17 @@ def scheme_status_filters
     }.freeze
   end
 
+  def location_status_filters
```
Comment (Airk0n) [inline app/helpers/filters_helper.rb:59]: I've not seen this before, is this shorthand for a switch statement? 
If it's doing what I think it is then I would assume there's a helper function that takes a string, removes replaces the underscores with spaces and capitalises the first letter?

File: app/models/location.rb
Comment (Airk0n) [inline app/models/location.rb:1]: Looks good

**Review by Airk0n — APPROVED** [review]
Looks good to me

File: app/helpers/filters_helper.rb (hunk 1 above)
Comment (kosiakkatrina) [reply app/helpers/filters_helper.rb:59]: Do you mean the hash? https://ruby-doc.org/core-2.7.0/Hash.html
We're just accessing values using keys, similarly like you'd access values in arrays using indexes

**Pull request merged**